From: A. Maitland Bottoms <bottoms@debian.org>
Subject: doxygen pdf

Go ahead and let Doxygen make a .pdf file for us.

--- a/host/docs/Doxyfile.in
+++ b/host/docs/Doxyfile.in
@@ -601,7 +601,7 @@
 # The QUIET tag can be used to turn on/off the messages that are generated
 # by doxygen. Possible values are YES and NO. If left blank NO is used.
 
-QUIET                  = YES
+QUIET                  = NO
 
 # The WARNINGS tag can be used to turn on/off the warning messages that are
 # generated by doxygen. Possible values are YES and NO. If left blank
@@ -1288,7 +1288,7 @@
 # If the GENERATE_LATEX tag is set to YES (the default) Doxygen will
 # generate Latex output.
 
-GENERATE_LATEX         = NO
+GENERATE_LATEX         = YES
 
 # The LATEX_OUTPUT tag is used to specify where the LaTeX docs will be put.
 # If a relative path is entered the value of OUTPUT_DIRECTORY will be
--- a/host/docs/CMakeLists.txt
+++ b/host/docs/CMakeLists.txt
@@ -109,6 +109,7 @@
     ADD_CUSTOM_COMMAND(
         OUTPUT ${CMAKE_CURRENT_BINARY_DIR_DOXYGEN} DEPENDS ${DOXYGEN_DEPENDENCIES}
         COMMAND ${DOXYGEN_EXECUTABLE} ${CMAKE_CURRENT_BINARY_DIR}/Doxyfile
+	COMMAND cd ${CMAKE_CURRENT_BINARY_DIR_DOXYGEN}/latex && make
         COMMENT "Generating documentation with doxygen"
     )
 
